src/main/java/hudson/plugins/jira/JiraBuildAction.java

```diff
@@ -11,11 +15,12 @@
  * @author Kohsuke Kawaguchi
  */
 public class JiraBuildAction implements Action {
-    public final AbstractBuild<?, ?> owner;
+    
+	public final Run<?, ?> owner;
```

Comment:
@warden @jan-zajic
I'm not aware about plugins utilizing this API as well. 
On the other hand I've definitely used this action within [Scriptler](https://wiki.jenkins-ci.org/display/JENKINS/Scriptler+Plugin) scripts at my previous work. Such valid usages of public API are also under risk. Not sure about my instance, I do not recall exact types and methods unfortunately.

> Also, I'm curious how do you know about RunAction2, that's good to know, but I don't see any description in the API doc for this. Is this just your Jenkins core source knowledge from exp? :)

Yep, just a sacred knowledge I've got during Workflow integration of other plugin. Documentation needs improvement, of course